server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -829,6 +829,10 @@ public IndicesQueryCache getIndicesQueryCache() {
         return indicesQueryCache;
     }
 
+    public MapperRegistry getMapperRegistry() {
+        return mapperRegistry;
+    }
```

Comment:
should we only expose the method that is needed? we already have getFieldFilter, wondering if we should follow the same approach.

Reply:
++, I pushed https://github.com/elastic/elasticsearch/pull/69977/commits/78fb83ed12c190e52aa24eca0cd8b814271f5bee